src/persistence/secondary/mongo-secondary-entity-manager.ts

```diff
@@ -0,0 +1,62 @@
+import { Entity, Model } from '@antjs/ant-js/build/ant';
+import { Collection, MongoClient } from 'mongodb';
+import { SecondaryEntityManager } from './secondary-entity-manager';
+
+export class MongoSecondaryEntityManager<TEntity extends Entity> implements SecondaryEntityManager<TEntity> {
+
+  model: Model<TEntity>;
+  private collection: Collection;
```

Comment:
Private attributes in JS must start with underscore (for the sake of readability).